static/app/components/events/interfaces/spans/spanTreeModel.tsx

```diff
@@ -772,6 +773,10 @@ class SpanTreeModel {
             false
           );
 
+          const startTimeDelta =
+            this.span.start_timestamp - parsedRootSpan.span.start_timestamp;
+          adjustEmbeddedTransactionTimestamps(parsedRootSpan, startTimeDelta);
+
```

Comment:
I'm making the assumption here that it is safe to assume that we should always adjust the timestamps for embedded transactions. Are there any cases where we wouldn't want this to happen?

Reply:
@0Calories I feel like we should always adjust whenever `startTimeDelta > 0` (i.e. `this.span.start_timestamp > parsedRootSpan.span.start_timestamp`).

We shouldn't *always* adjust for when `startTimeDelta < 0`; since the SDK may not capture all the spans prior to when the embedded transaction was created. We'll need to be careful for adjusting for this case.